controllers/csm_controller.go

```diff
@@ -895,22 +889,10 @@ func (r *ContainerStorageModuleReconciler) SyncCSM(ctx context.Context, cr csmv1
 		}
 
 	}
-	// If dell connectivity client is deployed, create role/rolebindings in the csm namespaces
-	if err = utils.CheckAccAndCreateOrDeleteRbac(ctx, operatorConfig, ctrlClient, false); err != nil {
-		return err
-	}
 
 	return nil
 }
 
-// SyncRbac - Sync the current installation - this can lead to a create or update
-func (r *ContainerStorageModuleReconciler) SyncRbac(ctx context.Context, _ csmv1.ContainerStorageModule, operatorConfig utils.OperatorConfig, ctrlClient client.Client) error {
```

Comment:
why are we removing this function?

Reply:
SyncRbac was used by the acc code. no longer needed.

Reply:
It was also used by CSM code though, right?
err := r.SyncRbac(ctx, *csm, *operatorConfig, r.Client)

Reply:
No, it's safe to remove. The CSM object wasn't used.